pkg/operator/operatorclient/informers.go

```diff
@@ -1,27 +1,18 @@
 package operatorclient
 
 import (
-	"time"
-
-	"k8s.io/client-go/informers"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
 	"k8s.io/client-go/kubernetes"
 )
 
-type KubeInformersForNamespaces map[string]informers.SharedInformerFactory
-
-func NewKubeInformersForNamespaces(kubeClient kubernetes.Interface) KubeInformersForNamespaces {
-	return map[string]informers.SharedInformerFactory{
-		"": informers.NewSharedInformerFactory(kubeClient, 10*time.Minute),
-		UserSpecifiedGlobalConfigNamespace:    informers.NewSharedInformerFactoryWithOptions(kubeClient, 10*time.Minute, informers.WithNamespace(UserSpecifiedGlobalConfigNamespace)),
-		MachineSpecifiedGlobalConfigNamespace: informers.NewSharedInformerFactoryWithOptions(kubeClient, 10*time.Minute, informers.WithNamespace(MachineSpecifiedGlobalConfigNamespace)),
-		OperatorNamespace:                     informers.NewSharedInformerFactoryWithOptions(kubeClient, 10*time.Minute, informers.WithNamespace(OperatorNamespace)),
-		TargetNamespaceName:                   informers.NewSharedInformerFactoryWithOptions(kubeClient, 10*time.Minute, informers.WithNamespace(TargetNamespaceName)),
-		"kube-system":                         informers.NewSharedInformerFactoryWithOptions(kubeClient, 10*time.Minute, informers.WithNamespace("kube-system")),
-	}
-}
-
-func (i KubeInformersForNamespaces) Start(stopCh <-chan struct{}) {
-	for _, informer := range i {
-		informer.Start(stopCh)
-	}
+func NewKubeInformersForNamespaces(kubeClient kubernetes.Interface) v1helpers.KubeInformersForNamespaces {
```

Comment:
I just updated https://github.com/openshift/cluster-kube-apiserver-operator/pull/208 with this

you should delete this